### ISSUE TITLE
Game crashed

### ISSUE THREAD

**Issue by XC-Zhang** [zlsa/atc#362]
Game crashed when playing at SAEZ. Following are related messages logged in browser's console:

```
Airline not found:sa-aal @aircraft.js:2237
Uncaught TypeError: Cannot read property 'chooseAircraft' of undefined @aircraft.js:2265
    aircraft_new    @ aircraft.js:2265
    spawnAircraft   @ airport.js:259
    game_update_pre @ game.js:239
    update          @ modules.js:265
```

**Comment by Fechulo** [comment]
I'll fix it now, that's due to the recent change where sa-aal was changed to aal/long

**Comment by Fechulo** [comment]
Should be fixed with #363
